### ISSUE TITLE
Rotate Proxy AUtomatically

### ISSUE THREAD

**Issue by permis20182018** [dragonfruitnetwork/onionfruit#27]
Hello , First thanks for the soft its working just fine 

I wonder if there is a way to auto reconnect to defirrent Proxy every XX Second or Minute using onionfruit 

Thx

**Comment by aspriddell** [comment]
do you mean changing ip addresses? the proxy itself doesn't need changing as it's a localhost one.

i'm also pretty sure that the Tor client itself does this automatically at set intervals

**Comment by aspriddell** [comment]
yes, that's entirely possible. I'll keep this open for progress updates

**Comment by permis20182018** [comment]
Thanks all the best
